dependencies/config/config.go

```diff
@@ -29,9 +29,13 @@ type Config struct {
 
 func init() {
 	configEnv := fmt.Sprintf("%sCONFIG", entry.DependencyPrefix)
-	if configDeps := env.SplitEnvToList(configEnv); len(configDeps) > 0 {
+	fmt.Println(configEnv)
+	separator := ":"
+	if strings.Contains(configEnv, separator) {
```

Comment:
@dkrzyszczyk consider using `:` literal instead of a variable. It's easier to understand single line and the variable is not used elsewhere.